packages/editor/src/components/post-publish-panel/prepublish.js

```diff
@@ -102,10 +102,20 @@ function PostPublishPanelPrepublish( { children } ) {
 			<p>{ prePublishBodyText }</p>
 			<div className="components-site-card">
 				{ siteIcon }
-				<div className="components-site-info">
-					<span className="components-site-name">{ siteTitle }</span>
-					<span className="components-site-home">{ siteHome }</span>
-				</div>
+				{ ( siteTitle || siteHome ) && (
```

Comment:
Also the individual checks that follow after should not be there

Reply:
And honestly, in terms of UI, these values should always be there or have a fallback to "(Untitled)" at most.

Reply:
Thanks for your feedback, folks! 🙇 

> And honestly, in terms of UI, these values should always be there or have a fallback to "(Untitled)" at most.

Considering we want the block to be always visible, I'd say this would be the minimal version of it, where none of the values are available:
<img width="275" alt="Screenshot 2021-04-14 at 14 01 01" src="https://user-images.githubusercontent.com/1451471/114707070-1287a980-9d2a-11eb-9561-f01d3e93f3b2.png">

I haven't put anything in place of the `siteHome` (URL) - I think it's OK to just leave it empty. What do you think?

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/30812/commits/3a857532f113b5f9a95e7c707ce931963a04090d